kyo-core/shared/src/main/scala/kyo/options.scala

```diff
@@ -1,11 +1,16 @@
 package kyo
 
-type Options = Aborts[None.type]
+type Options >: Options.Effects <: Options.Effects
 
 object Options:
 
+    private class Empty
```

Comment:
I've added a separate type to abort with since users can use `Aborts.fail(None)`, which could conflict with `Options`. This is unrelated to the opaque type change.

Reply:
I don't mind if `Aborts.fail(None) == Options.empty`...

Reply:
Thinking it through, the change to `Empty` is actually necessary if the effect is `opaque`. Example:

```scala
val a: Either[None, Int] < Options = Aborts.run[None.type](Options.get(Option.empty[Int]))
val b: Option[Either[None, Int]] = Options.run(a)
```

The `Aborts.run` would handle `Options` since the tag will match but keep it pending since it's `opaque`. If it's not `opaque`, that's not an issue. I'm not sure there's much benefit in keeping `Options` as `opaque`, though. It'd allow us to evolve the effect with different internal encodings, even not based on `Aborts`, but it doesn't seem very likely. I'll revert it.